tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -30,58 +31,17 @@ public DotNetCoreSampleAppsTest(ITestOutputHelper output) : base(output)
         private readonly string SdkVersionMessageFormat = "Using .NET Core SDK Version: {0}";
 
         [Fact]
-        public void Builds_NetCore10App_UsingNetCore11_DotNetSdkVersion()
```

Comment:
Discard the changes to the tests in this file

Reply:
do we still want this test?

Reply:
yes we still want this test...after my checked in changes, this test will pass...so that is why we want all changes here to be discarded as stuff should work fine after my changes..